src/Library/demos/Column View/main.js

```diff
@@ -0,0 +1,123 @@
+import Gtk from "gi://Gtk";
+import Gio from "gi://Gio";
+import GObject from "gi://GObject";
+
+const column_view = workbench.builder.get_object("column_view");
+const col1 = workbench.builder.get_object("col1");
+const col2 = workbench.builder.get_object("col2");
+const col3 = workbench.builder.get_object("col3");
+
+function createFirstCol() {
```

Comment:
Let's remove all the function wrappers on these, and add comments like:

```
// View

// View Column 1
...

// View Column 2
...
```

Under each column comment, we will define the factory, then the sorter, since these are all the "view" part of the model-view-controller (there's no controller in this demo though and that's fine).